XavierGuertin/WildfireVisualizationProject
# #51 - feat(frontend): Added Buttons to UI

### Summary

Added three buttons (Settings, Language, Reset) with dropdown buttons to the UI.

### Changes Made

- Added "Settings", "Language", "Reset" buttons with dropdown buttons to frontend
- Resized and removed backgrounds to images, placed in dropdown buttons and buttons
- Added mouse handler to close dropdowns when clicking the buttons twice or outside the buttons
- Added active handling to buttons, white border appears if clicked

### Testing Instructions

1. Click the "Language" image button (middle button in top-left) to have the dropdown of languages appear. Confirm the button's border becomes white.
2. Click on any language, and click "OK" in the alert box.
3. Click the "Language" image button (middle button in top-left) to have the dropdown of languages appear. Confirm the checkmark appears to the left of the correct language.
4. Click elsewhere other than on the "Language" button and its dropdown, confirm the dropdown and the button's white border disappears.
("Reset" button, Steps in order: 1, 2, 1, 4)

### Screenshots (if applicable)

![image](https://github.com/user-attachments/assets/fd60e524-8299-4970-b126-f773444dd1d6)
![image](https://github.com/user-attachments/assets/09a255fa-410d-4ae3-9969-c2bca644bc1a)
![image](https://github.com/user-attachments/assets/7ab45945-6160-4660-9f91-0f59de73c2e1)

https://github.com/user-attachments/assets/d78c145d-e6e4-4f55-a290-1800a3464a81



### Additional Notes

Beginning of work on backend, missing tilemap reference. The background(map) in the screenshots above do not appear as they will.
